ipatests/pytest_ipa/integration/tasks.py

```diff
@@ -2039,6 +2039,8 @@ def install_packages(host, pkgs):
     # Only supports RHEL 8+ and Fedora for now
     if platform in ('rhel', 'fedora'):
         install_cmd = ['/usr/bin/dnf', 'install', '-y']
+    elif platform in ('ubuntu'):
+        install_cmd = ['apt-get']
```

Comment:
This line should be `install_cmd = ['apt-get', 'install', '-y']`

Reply:
updated